test/Microsoft.ML.TestFramework/Attributes/NotCentOS7FactAttribute.cs

```diff
@@ -19,6 +19,12 @@ public NotCentOS7FactAttribute() : base("These tests are not CentOS7 compliant."
         }
         protected override bool IsEnvironmentSupported()
         {
+            // Featurizers.dll must exist
+            if (!Microsoft.ML.TestFrameworkCommon.Utility.NativeLibrary.NativeLibraryExists("Featurizers"))
```

Comment:
This doesn't seem right. What is this for?

Reply:
Makes sense, I'll create a new attribute. I did it this way because this attribute was created specifically for the Featurizers, but was named more generically than that. But yeah, since its more generic I'll create a new one so things are more clear.

Reply:
Alright, new attribute is created and pushed.